nan.h

```diff
@@ -1654,12 +1661,12 @@ class Callback {
       Execute(progress);
   }
 
-  void SendProgress_(const char *data, size_t size) {
-    char *new_data = new char[size];
+  void SendProgress_(T *data, size_t size) {
+    T *new_data = new T[size];
     memcpy(new_data, data, size);
```

Comment:
I will likely have questions about this at some later point, if google doesn't yield enough information. I assume trying to compile with C++03 will break it, hence makes it testable?